vnext/Microsoft.ReactNative/IReactContext.h

```diff
@@ -6,14 +6,15 @@
 #include "ReactHost/React.h"
 #include "winrt/Microsoft.ReactNative.h"
 
-namespace winrt::Microsoft::ReactNative {
+namespace winrt::Microsoft::ReactNative::implementation {
```

Comment:
>implementation [](start = 41, length = 14)

(nit) is that a common pattern?

Reply:
Yes, the C++/WinRT usually puts the implementation in the winrt::Microsoft::ReactNative::implementation namespace.
Previously here I did not do it, just to make the namespace shorter, but now we have another ReactContext struct in the winrt::Microsoft::ReactNative namespace coming from the MS.RN.Cxx project and we must change the namespace here to avoid the naming conflict.

---
In reply to: [430532708](https://github.com/microsoft/react-native-windows/pull/5007#discussion_r430532708) [](ancestors = 430532708)